### ISSUE TITLE
Cannot read property 'balance' of undefined

### ISSUE THREAD

**Issue by kumavis** [ethereum/node-ethereum#43]
This is a result of an rpc call. Really the issue is that we should handle this case and throw a non-generic error.
```
TypeError: Cannot read property 'balance' of undefined
   at module.exports [as runCall] (/Users/kumavis/Development/Node/vapor/node_modules/node-ethereum/node_modules/ethereumjs-lib/lib/vm/runCall.js:36:54)
   at EthRPC.eth_call (/Users/kumavis/Development/Node/vapor/node_modules/node-ethereum/lib/ethRPC.js:252:6)
   at EthRPC.processRpc (/Users/kumavis/Development/Node/vapor/node_modules/node-ethereum/lib/ethRPC.js:28:6)
   at /Users/kumavis/Development/Node/vapor/node_modules/node-ethereum/lib/xhrRPC.js:42:11
   at Layer.handle [as handle_request] (/Users/kumavis/Development/Node/vapor/node_modules/node-ethereum/node_modules/express/lib/router/layer.js:82:5)
   at next (/Users/kumavis/Development/Node/vapor/node_modules/node-ethereum/node_modules/express/lib/router/route.js:110:13)
   at Route.dispatch (/Users/kumavis/Development/Node/vapor/node_modules/node-ethereum/node_modules/express/lib/router/route.js:91:3)
   at Layer.handle [as handle_request] (/Users/kumavis/Development/Node/vapor/node_modules/node-ethereum/node_modules/express/lib/router/layer.js:82:5)
   at /Users/kumavis/Development/Node/vapor/node_modules/node-ethereum/node_modules/express/lib/router/index.js:267:22
   at Function.proto.process_params (/Users/kumavis/Development/Node/vapor/node_modules/node-ethereum/node_modules/express/lib/router/index.js:321:12)
```

**Comment by wanderer** [comment]
@kumavis is this still a problem?

**Comment by kumavis** [comment]
@wanderer i didnt file this issue with enough context. I think it was about calling a contract that doesnt exist or getting the balance of an account that doesnt exist.

**Comment by wanderer** [comment]
ok so that was acutely  a bug. fixed in #6b61876
